File: Spring Boot/Courzelo/src/main/java/org/example/courzelo/dto/responses/GradeResponse.java
package org.example.courzelo.dto.responses;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class GradeResponse {
    private String id;
    private String name;
    private String moduleID;
    private String moduleName;
    private Double scoreToPass;
    private String institutionID;
    private boolean valid;
    private String groupID;
    private String studentEmail;
    private double grade;
}

File: Angular/src/app/shared/models/institution/GradeResponse.ts
export interface GradeResponse {
    id: string;
    name: string;
    moduleID: string;
    moduleName: string;
    scoreToPass: number;
    groupID: string;
    institutionID: string;
    studentEmail: string;
    grade: number;
    average: number;
    valid: boolean;
}
